src/System.IO.Compression/src/System/IO/Compression/Deflater.cs

```diff
@@ -251,5 +252,5 @@ private ZErrorCode Deflate(ZFlushCode flushCode)
         }
         #endregion  // helpers & native call wrappers
 
-    }  // internal class DeflaterZLib
+    }  // internal class Deflater
```

Comment:
Do we usually have these end of class and namespace comments?  It seems something that the IDE can do with "CTRL+}".